src/modules/Dropdown/DropdownSearchInput.js

```diff
@@ -18,29 +13,27 @@ class DropdownSearchInput extends Component {
     /** An element type to render as (string or function). */
     as: customPropTypes.as,
 
+    /** An input can have the auto complete. */
+    autoComplete: PropTypes.string,
```

Comment:
As I remember, there can be `on` and `off` values.

https://www.w3schools.com/tags/att_input_autocomplete.asp